client/me/purchases/billing-history/receipt.tsx

```diff
@@ -564,14 +565,19 @@ function ReceiptLineItem( {
 		isSmallestUnit: true,
 		stripZeros: true,
 	} );
+
+	const isSitelessDomain = /^siteless\.(agencies\.automattic|a4a)\.com/.test( item.domain );
```

Comment:
Minor but now that this regex test is showing up in multiple places, I wonder if this should become a helper util also?

Reply:
Yeah I thought about that, but the regex patterns are actually all different. This tests just for a4a, another tests for marketplace or a4a, and another tests for those and other services. 🤔